analytic_engine/src/sst/parquet/meta_data.rs

```diff
@@ -38,68 +45,125 @@ pub enum Error {
 
 define_result!(Error);
 
+const DEFAULT_BLOOM_FILTER_VERSION: u32 = 0;
+
+#[derive(Debug, Clone, PartialEq, Eq, Default)]
+pub struct RowGroupBloomFilter {
+    // The column filter can be None if the column is not indexed.
+    column_filters: Vec<Option<Bloom>>,
```

Comment:
I think another issue for using col_idx to refer column, it will block us to implement  some DDL, such as add column after existing one.

Reply:
I will open an issue to track this.

Reply:
https://github.com/CeresDB/ceresdb/issues/616